README.md

```diff
@@ -420,6 +420,82 @@ The local key helper is responsible for:
 - Producing signatures with the binding key
 - Cleanup of the binding key and its artifacts (when the user clears the browser session or the key is unused for a long time)
 
+### Local key helper on Mac
+
+On macOS, a Local Key Helper is a protocol (`LocalKeyHelper.h`), and its implementation (`LocalKeyHelperImpl.h`/`LocalKeyHelperImpl.m`) is yet to be determined.
+
+The Local Key Helper will be implemented and shipped as an XPC service on macOS. It will be signed with app sandbox entitlement for securety requirement.
+
+To start the Local Key Helper XPC service, the service plist (property list) needs to be defined and registered in the launchAgent. This will ensure that the service starts up after each user's system signing session.
+
+Here is an example of the Local Key Helper XPC service plist:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>com.contoso.LocalKeyHelper</string>
+    <key>Program</key>
+    <string>/pathToXcpService/LocalKepHelper.xpc/Contents/MacOS/LocalKepHelper</string>
+    <key>RunAtLoad</key>
```

Comment:
Always running can result in wasting resources. Can we run on demand, on first usage, and shutdown after 5 minutes?

Reply:
Discussed offline. The loading pattern can be flexible, and will not define the option here since not all users/devices need the XPC service to run at load.

Reply:
So will we remove the `RunAtLoad` option above?

Reply:
It is up to the developer of the XPC service to decide what to do. Above is just a sample and not a prescribed way to build an XPC service.

Reply:
So, we can keep as is